library/pod-security-policy/apparmor/src.rego

```diff
@@ -0,0 +1,16 @@
+package k8spspapparmor
+
+violation[{"msg": msg, "details": {}}] {
+    metadata := input.review.object.metadata
+    not input_apparmor_allowed(metadata)
+    msg := sprintf("apparmor profile is not allowed, pod: %v. Allowed profiles: %v", [input.review.object.metadata.name, input.parameters.allowedProfiles])
+}
+
+input_apparmor_allowed(metadata) {
+    metadata.annotations
+    not metadata.annotations["apparmor.security.beta.kubernetes.io/pod"]
```

Comment:
should we use `container.apparmor.security.beta.kubernetes.io`? https://kubernetes.io/docs/tutorials/clusters/apparmor/#securing-a-pod

Reply:
this PR is on the pod level apparmor, I can add it to this or do a follow up?

Reply:
IMO if we don't take care of the container level, this PR is not really complete as it does not mimic what users might expect from this PSP. WDYT?

Reply:
thanks! added container level implementation, supports both pod and container level now